tech/languages/nodejs/nodejs.md

```diff
@@ -4,36 +4,45 @@ subsection: nodejs
 section: tech-languages
 order: 1
 version: LTS
+description: Open-source cross-platform server-side JavaScript runtime environment.
 ---
 
 # Node.js
 
-## Node.js and npm
+## Node.js and npm installation
 
 Starting from Fedora 24, npm is a part of Node.js package and does not need to be installed separately. Therefore, to install both npm and Node.js, you need to run:
 
 ```
 $ sudo dnf install nodejs
 ```
 
-In case you are running Fedora 23, you should run:
+This will install V8 Javascript Engine, Node.js runtime and npm package manager and their dependencies. Versions present in repositories are usually current or to-be (in Rawhide) Node.js LTS releases, including npm and V8 engine that come with them.
+
+You can check the available versions with:
 
 ```
-$ sudo dnf install npm
+$ dnf info nodejs
+$ dnf info nodejs --enablerepo rawhide
```

Comment:
Rawhide repository is not included in basic Fedora install AFAIK, so this wouldn't work.

Reply:
This will soon work with just --releasever=30 instead of  `--enablerepo rawhide`, right?